themes/package.json

```diff
@@ -0,0 +1,13 @@
+{
+	"name": "remnote-ctp-port-personal",
```

Comment:
Looks like you forgot to change a few lines from your personal repo to the catppuccin repo :P

Reply:
:)